app/views/shared/_nav_auth.html.slim

```diff
@@ -12,7 +12,10 @@ nav.bg-white
           = link_to t('shared.nav_auth.news'), root_path, class: 'mr3 gray'
           = link_to t('shared.nav_auth.contact'), root_path, class: 'mr3 gray'
       .sm-col-right.sm-right-align
-        div = t('shared.nav_auth.welcome', email: current_user.email)
+        div
+          <strong>
```

Comment:
I think we've generally been using classes for bolding, so something like this could condense it a bit:

```
      .sm-col-right.sm-right-align
        = t('shared.nav_auth.welcome')
        '
        span.bold = current_user.email
```